docs/building_the_project_with_adaptivecpp.rst

```diff
@@ -35,7 +35,7 @@ On Linux (other OSes are not supported with the AdaptiveCpp compiler):
 
   # Inside <path to onemkl>
```

Comment:
This file still says `<path to onemkl>` (meaning this project) here and in the "Environment Setup" above

Reply:
Fixed in https://github.com/oneapi-src/oneMKL/pull/602/commits/a42a97089dae9a1e672a1560b0bc4545cea64cd9